spec/fixtures/files/not_parseable.yml

```diff
@@ -0,0 +1,2 @@
+"Title":
+  - `description`
```

Comment:
@LukeDDC You could add a comment to explain why this line is not parsable

Reply:
@rodrigomasaru Message added.